app/assets/javascripts/controllers/ownership/ownership_form_controller.js

```diff
@@ -0,0 +1,77 @@
+ManageIQ.angular.app.controller('ownershipFormController', ['$http', '$scope', 'objectIds', 'miqService', function($http, $scope, objectIds, miqService) {
+  var init = function() {
+    $scope.ownershipModel = {
+      user: '',
+      group: ''
+    };
+    $scope.afterGet  = false;
+    $scope.newRecord = false;
+    $scope.modelCopy = angular.copy( $scope.ownershipModel );
+    $scope.model     = "ownershipModel";
+    $scope.objectIds = objectIds;
+    $scope.saveable = miqService.saveable;
+
+    ManageIQ.angular.scope = $scope;
+
+    miqService.sparkleOn();
+    $http.get('ownership_form_fields/' + objectIds).success(function(data) {
+      $scope.ownershipModel.user = data.user;
+      $scope.ownershipModel.group = data.group;
+      $scope.afterGet = true;
+      $scope.modelCopy = angular.copy( $scope.ownershipModel );
+      miqService.sparkleOff();
+    });
+  };
+
+  $scope.canValidateBasicInfo = function () {
+    if ($scope.isBasicInfoValid())
```

Comment:
I know you're following a pattern from the other controllers, but I'd like to fix this so that we don't repeat ourselves too much.. but for now, this should probably be `$scope.canValidateBasicInfo = $scope.isBasicInfoValid;`...

And for `isBasicInfoValid`, you could just return the conditon, and skip that `if (bool) return true; else return false;`..